pitch.md

```diff
@@ -7,14 +7,28 @@ _Who are you helping, with your language? What is that person's
 experience like now? How would their experience be better if you could help 
 them?_
 
+My language is for people who want to draw tableau trees, more specifically for students in CS 81.
+CS 81 requires trees that are rendered digitally and the recommended method for drawing the trees is to use Google Sheets.
+So students have to create text-boxes and arrows and drag them around to draw tree, which is quite cumbersome.
+A common alternative is to use LaTeX to typeset a tree.
+However, not everyone knows how to use LaTeX because the syntax is a bit challenging (and requires a lot of boilerplate). 
+Even if someone uses LaTeX for their other classes, they may not be familiar with the LaTeX packages for typesetting trees.
 
 ## Why a language?
 _Why is a **DSL** appropriate for your user(s)? How does it address the need?_
 
+Logicians already use a DSL, LaTeX, for this purpose.
+A language has a steeper learning curve compared to using Google Sheets, but once someone has learned a language, it should be much easier to use the language than drawing a diagram.
+It seems like another DSL, with a simpler interface compared to LaTeX can improve a user's experience.
 
 ## Why you?
 _What excites you about this idea? How did you come up with it?_
 
+I'm taking CS 81 and I really dislike drawing trees on Google Sheets.
+I use LaTeX for my other classes, so my initial thought was to use it for CS 81 as well.
+I tried using several LaTeX packages to draw tableaux trees but my experience has been frustrating thus far.
+I TeX-ed tableau trees for one of my assignments but it required a lot of trial and error as well as reading a _lot_ of samples of various tree structures. 
+I would be really happy if there was an easier way for me to draw trees for my homework.
 
 ## Interface (syntax)
 _What kinds of things can the user say in your language. How, generally, do they
```

Comment:
"How might your experience augment your partner's thoughts on the domain?" Even though I don't know much about tableau trees or LaTeX, I think I can represent a user who needs a very simple, intuitive language -- a user who might only use the minimum features in this DSL. I represent the user who wants to avoid having to learn features that are unnecessary to the task of creating the most basic form of a tableau tree.